src/sites/image/fopkodiak.site.js

```diff
@@ -0,0 +1,10 @@
+_.register({
+  rule: {
+    host: /^fopkodiak\.site$/,
+    path: /image\//,
```

Comment:
```suggestion
    path: /^\/image\//,
```
Probably always starts with `/image/`

Reply:
This page does not have pop-up or countdown.
Do we need this?

Reply:
for me, i say add it into the script.